website/docs/r/windows_web_app.html.markdown

```diff
@@ -179,6 +179,7 @@ ASP.NET V4.8 | v4.0
 .NET 7 (STS) | v7.0
```

Comment:
.NET 7.0 (STS) no longer supported on Azure Web apps:

<img width="582" height="348" alt="Image" src="https://github.com/user-attachments/assets/2e2754fa-ca07-4e29-be51-ae4f98ffa7be" />

I would suggest this should be removed.

Reply:
I'm not sure what the route to deprecation is here, v6 is also still in the provider but not available either.

Reply:
Maybe making some cleanup is not as urgent as adding this new version.

And what would be the impact of removing older versions on running webapps / functions / else ?

Reply:
I understand.  I suggest my comment be ignored then.  Better that this doesn't become a breaking change